src/EditorFeatures/CSharpTest/PdbSourceDocument/AbstractPdbSourceDocumentTests.cs

```diff
@@ -159,6 +160,13 @@ protected static async Task RunTestAsync(Func<string, Task> testRunner)
 
                 Assert.Equal(document.FilePath, file.FilePath);
 
+                // Mapping the project from the generated document should map back to the original project
+                var exportProvider = (IMefHostExportProvider)project.Solution.Workspace.Services.HostServices;
+                var provider = exportProvider.GetExportedValues<IMetadataAsSourceFileProvider>().OfType<PdbSourceDocumentMetadataAsSourceFileProvider>().Single();
+                var mappedProject = provider.MapDocument(document);
+                Assert.NotNull(mappedProject);
+                Assert.Equal(project.Id, mappedProject!.Id);
```

Comment:
No new tests, but this adds an extra verification to all tests in `PdbSourceDocumentTests.cs`